src/tracks/playabletrack/wavetrack/ui/SampleHandle.cpp

```diff
@@ -240,9 +254,9 @@ UIHandle::Result SampleHandle::Click
       Floats newSampleRegion{ 1 + 2 * (size_t)SMOOTHING_BRUSH_RADIUS };
 
       //Get a sample  from the track to do some tricks on.
-      mClickedTrack->GetFloats(sampleRegion.get(),
```

Comment:
I'm looking at what did not change in these routines, as well as what did.

There are sampleCount members of SampleHandle.h and the track's LongSamplesToTime is used to quantize the time value to a discrete sample count, by the WaveTrack's rate.

I think that should no longer happen -- instead it should always quantize according to a clip's rate and stretch.